File: src/main/java/kaalka/KaalkaUtils.java
package kaalka;

import java.text.SimpleDateFormat;
import java.util.*;

/**
 * KaalkaUtils: Utility functions for UTC time handling
 */
public class KaalkaUtils {
    public static String utcNow() {
        return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").format(new Date());
    }
    public static Date utcNowDate() {
        return new Date();
    }
    public static Date parseUtc(String ts) throws Exception {
        return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").parse(ts);
    }
}

File: src/main/kotlin/kaalka/KaalkaUtils.kt
package kaalka

import java.text.SimpleDateFormat
import java.util.*

/**
 * Utility functions for UTC time handling
 */
object KaalkaUtils {
    fun utcNow(): String = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").format(Date())
    fun parseUtc(ts: String): Date = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").parse(ts)
}
